docs/platform/howto/byoc/create-custom-cloud.rst

```diff
@@ -0,0 +1,146 @@
+Create a custom cloud in Aiven |beta|
+=====================================
+
+A custom cloud is your own cloud infarstructre integrated with your Aiven organization (as opposed to an Aiven-managed infrastructure). Using a custom cloud in Aiven may be the optimal solution if you have specific business needs or project requirements, such as a strict regulatory compliance. With a custom cloud, you can manage your own infarstructure based on your expectations and tailor it to your budget.
+
+Learn how to add a :doc:`custom cloud </docs/platform/concepts/byoc>` to your Aiven organization, and check out how it works.
+
+.. important::
+
+   * To create a custom cloud in your Aiven organization, you need the bring your own cloud (BYOC) feature enabled for this organization.
+   * BYOC is a beta feature, and its availability is limited. To enable BYOC for your Aiven organization, contact `sales@Aiven.io <mailto:sales@Aiven.io>`_.
+   * For more information on BYOC, see :doc:`Bring your own cloud </docs/platform/concepts/byoc>`. 
+
+About creating a custom cloud
+-----------------------------
+
+If you enabled BYOC and you have the administrator's role in your organization, you can create a custom cloud using `Aiven Console <https://console.aiven.io/>`_.
+
+Limitations
+'''''''''''
+
+* Administrator's role is required for creating custom clouds.
+* BYOC beta version supports the AWS cloud provider only.
+* You need to use the :ref:`BYOC standard deployment <byoc-standard>` as a deployment model for your custom cloud.
+
+Prerequisites
+-------------
+
+* Administrator's role for your Aiven organization
+* BYOC feature enabled from `sales@Aiven.io <mailto:sales@Aiven.io>`_
+* Access to `Aiven Console <https://console.aiven.io/>`_
+
+Create a custom cloud
+---------------------
+
```

Comment:
As the following steps are related to AWS only, should we add a subtitle "AWS"? That way when we have GCP and Azure ready it would be easier to read and understand where you are?

Reply:
@Matthieu68857 , this is a valid comment, and I think in the future we'd need to provide the subsections and subtitles per cloud provider. Now, when we support just one, this would mean that inside the "Create a custom cloud" section is one subsection only ("AWS"), which is not a good practice. Normally, we build subsections if there are at least two logical elements to be included in the upper section. For now, based on your comment, we have a pretty good idea on how it should look like in the future.